### PR TITLE
[repo-ci-improvement] : add release-drafter to ccm

### PR DESCRIPTION
**Pull request by rahulait**
### General:
Adding release-drafter to CCM repo
* [x] Have you removed all sensitive information, including but not limited to access keys and passwords?
* [x] Have you checked to ensure there aren't other open or closed [Pull Requests](../../pulls) for the same bug/feature/question?

### Pull Request Guidelines:

1. [x] Does your submission pass tests?
1. [ ] Have you added tests? 
1. [x] Are you addressing a single feature in this PR? 
1. [x] Are your commits atomic, addressing one change per commit?
1. [x] Are you following the conventions of the language? 
1. [x] Have you saved your large formatting changes for a different PR, so we can focus on your work?
1. [ ] Have you explained your rationale for why this feature is needed? 
1. [ ] Have you linked your PR to an [open issue](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)

### CODE REVIEW

File: .github/workflows/pr-labeler.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Label PR
+        uses: release-drafter/release-drafter@v6
+        with:
+          disable-releaser: github.ref != 'refs/heads/main'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (komer3) [inline .github/workflows/pr-labeler.yml:25]: nit: add new line at end of file

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  update_release_draft:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: release-drafter/release-drafter@v6
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (komer3) [inline .github/workflows/release-drafter.yml:21]: Same here :)

File: .github/workflows/pr-labeler.yml (hunk 1 above)
Comment (rahulait) [reply .github/workflows/pr-labeler.yml:25]: whoops, somehow missed it

Comment (rahulait) [reply .github/workflows/pr-labeler.yml]: should be all set now

File: .github/workflows/release-drafter.yml (hunk 2 above)
Comment (rahulait) [reply .github/workflows/release-drafter.yml]: updated

**Review by komer3 — APPROVED** [review]
LGTM!

**Pull request merged**